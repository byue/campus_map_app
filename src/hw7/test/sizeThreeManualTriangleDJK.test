# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 3 connected triangle graph.

CreateGraph g

AddNode g a
AddNode g b
AddNode g c
AddNode g d

AddEdge g a b 3.512442
AddEdge g b c 2.41
AddEdge g a c 9.41

FindPath g a c