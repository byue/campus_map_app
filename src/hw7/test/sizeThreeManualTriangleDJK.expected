# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 3 connected triangle graph.

created graph g

added node a to g
added node b to g
added node c to g
added node d to g

added edge 3.512 from a to b in g
added edge 2.410 from b to c in g
added edge 9.410 from a to c in g

path from a to c:
a to b with weight 3.512
b to c with weight 2.410
total cost: 5.922